project/AppDependencies.scala

```diff
@@ -31,5 +32,21 @@ object AppDependencies {
     "org.mockito" % "mockito-core" % "2.27.0" % "test"
   )
 
-  def apply(): Seq[ModuleID] = compile ++ test
+  private lazy val jettyVersion = "9.2.26.v20180806"
+  val overrides: Set[ModuleID] = Set(
```

Comment:
Wiremock and Selenium both have transitive dependencies on different versions of `jetty-http`. The latest version causes `ClassNotFoundException` so I've overridden it with a slightly older version.

Reply:
I'm introducing Wiremock as it appears we currently test the connector layer using a mocked out HTTP Client. 
To properly test all the JSON Serialization etc works I think we should use WireMock